go/teams/teams.go

```diff
@@ -397,9 +397,11 @@ func (t *Team) Rotate(ctx context.Context) error {
 	// load an empty member set (no membership changes)
 	memSet := newMemberSet()
 
-	admin, err := t.getAdminPermission(ctx, false)
+	// Try to get the admin perms if they are available, if not, proceed anyway
+	var admin *SCTeamAdmin
+	admin, err = t.getAdminPermission(ctx)
 	if err != nil {
-		return err
+		admin = nil
```

Comment:
Log the error for the next person to debug this. Maybe they will try to combine SBS, subteams, and writer rotates and it will just be too much.